src/commands/configuration/config-subcommands/view.ts

```diff
@@ -0,0 +1,197 @@
+/* 
+ * MIT License
+ * 
+ * Copyright (c) 2023-present Mirage Aegis
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import {
+    SlashCommandSubcommandBuilder, EmbedBuilder, ChatInputCommandInteraction, Snowflake
+} from "discord.js";
+import { Subcommand } from "../../../util/command-template.js";
+import { defaultErrorHandler } from "../../../util/error-handler.js";
+import {
+    GoLive, Greeting, Logs, ReactionRoles, Server, Shoutout
+} from "../../../schemas/server";
+import { AZURE } from "../../../util/colours.js";
+import { NEW_LINE_RE } from "../../../events/shoutout.js";
+
+/*
+ * Displays server configurations.
+ * The data displayed by this command includes:
+ * - the logs channels;
+ * - the go-live channel and message;
+ * - the shout out channel, role, and message;
+ * - the number of reaction roles;
+ * - the welcome channel and message; and
+ * - the goodbye channel and message.
+ */
+
+const name: string = "view";
+
+export const command: Subcommand = {
```

Comment:
This is the `/config view` command that displays server configurations to server administrators.